### DISCUSSION TITLE
Strange $PROFILE path when running as admin

### DISCUSSION THREAD

**Discussion by ghost** [PowerShell/PowerShell#15212] [Q&A]
I opened Powershell 7.1.3 as admin and checked profile paths to find that the start of it is being cut off for some reason:
![image](https://user-images.githubusercontent.com/41576511/114537460-222fc100-9c4a-11eb-961d-026c76344f4f.png)
Has anyone else experienced this?

**Comment by ninmonkey** [comment]
Does running as normal and admin using `pwsh -nop` then `$PROFILE | select *` have different values?

What do you get for
```ps1
$PROFILE.psobject.properties | %{   
  @(                                
    $_.Name                         
    $_.value                        
    $_.value | Test-Path            
  ) | Join-String -sep "`n" -op "`n"
}                                   
```
